test/compilable/testcstuff2.c

```diff
@@ -400,3 +400,11 @@ int test22245()
 }
 
 /***************************************************/
+// https://issues.dlang.org/show_bug.cgi?id=22265
+
+int test22265(const char *buf)
```

Comment:
Hmm, what about `const char * const buf` ? Which should produce an error?

Reply:
Both `char * const` and `const char * const` produces an error.  Now that you've highlighted this line, we don't actually support `char * const`, because this should compile instead:
```
int test22265(char * const buf)
{
    return ++(*buf);
}